### PR TITLE
MMF-941 Unified fixed-width layout

### PR DESCRIPTION
**Pull request by stas-vilchik**

### CODE REVIEW

File: server/sonar-web/pom.xml [hunk 1]
```diff
@@ -230,7 +230,7 @@
         <dependency>
           <groupId>com.sonarsource</groupId>
           <artifactId>sonarsource-branding</artifactId>
-          <version>1.3.0.304</version>
+          <version>1.3.0.307</version>
```
Comment (gregaubert) [inline server/sonar-web/pom.xml]: What is this ?

File: server/sonar-web/src/main/js/components/ui/OrganizationIcon.js [hunk 2]
```diff
@@ -20,10 +20,17 @@
 // @flow
 import React from 'react';
 
-export default function OrganizationIcon() {
+type Props = { className?: string, size?: number };
+
+export default function OrganizationIcon({ className, size = 16 }: Props) {
```
Comment (gregaubert) [inline server/sonar-web/src/main/js/components/ui/OrganizationIcon.js]: Could we move it to `js/components/icons-components/` folder ?

File: server/sonar-web/pom.xml (hunk 1 above)
Comment (stas-vilchik) [inline server/sonar-web/pom.xml]: https://github.com/SonarSource/sonarsource-branding

It contains SonarQube logo and the "official" footer.

Comment (gregaubert) [inline server/sonar-web/pom.xml]: Ah ok, and it's because you updated the logo on sonarsource-branding. Got it, thanks!

**Pull request merged**